source/includes/steps-fle-convert-to-a-remote-master-key.yaml

```diff
@@ -128,6 +139,17 @@ content: |
             const base64DataKeyId = key.toString('base64');
             console.log("DataKeyId [base64]: ", base64DataKeyId);
            });
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+
+           connectionString = "mongodb://localhost:27017";
```

Comment:
Could you use snake case here to stay consistent with the rest of the Python variable names?